aep/general/0151/aep.md.j2

```diff
@@ -75,10 +75,18 @@ but is not obligated to do so:
 
 - Resources that accept multiple parallel requests **may** place them in a
   queue rather than work on the requests simultaneously.
-- Resource that does not permit multiple requests in parallel (denying any new
-  request until the one that is in progress finishes) **must** return
-  `409 Conflict` if a user attempts a parallel request, and include an error
-  message explaining the situation.
+
+{% tab proto %}
+
+- A resource that does not support concurrent mutation requests **must** return an
+  `ABORTED` error if the resource has an outstanding mutation.
+
+{% tab oas %}
+
+- A resource that does not support concurrent mutation requests **must** respond
+  with a `409 Conflict` status code if the resource has an outstanding mutation.
+
+{% endtabs %}
```

Comment:
```suggestion
- Resource that does not permit multiple requests in parallel (denying any new
  request until the one that is in progress finishes) **must** return
  `ABORTED`/`409 Conflict` if a user attempts a parallel request, and include an error
  message explaining the situation.
```

We discussed this live - seems unfortunate to add all the tab chrome to the page just to vary a single term.  Thoughts?

Reply:
I'm fine with consolidating it. 

@wora gave feedback on the "include an error message" guidance earlier, and suggested that adding this type of guidance may result in users using the error string as a way to figure out the actual error (concurrent request). I removed that part intentionally based on that feedback (as I also agreed). Thoughts on that?